product-service/handlers/getProductsById.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+import { mockAsyncCall } from '../mocks/async.mock';
+import { productsMockData } from '../mocks/products.mock';
+
+export const getProductsById = async (event) => {
+    console.log('[Get Products List By ID] request, event:', event);
+
+    const headers = {
+        'Access-Control-Allow-Origin': '*',
+        'Access-Control-Allow-Credentials': true,
+    };
+
+    const productsData = await mockAsyncCall(productsMockData);
+    let productId = null;
+
+    if(event && event.pathParameters && event.pathParameters.productId != null){
+        productId = event.pathParameters.productId;
+    } else{
+        return {
+            statusCode: 400,
```

Comment:
I would suggest to move status codes to constant file

Reply:
Created constant file as requested